api-reference/beta/api/alert_list.md

```diff
@@ -0,0 +1,123 @@
+# List alerts
+
+Retrieve a list of [alert](../resources/alert.md) objects.
+
+## Permissions
+
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](../../../concepts/permissions_reference.md).
+
+|Permission type      | Permissions (from least to most privileged)              |
+|:--------------------|:---------------------------------------------------------|
+|Delegated (work or school account) |  SecurityEvents.Read.All, SecurityEvents.ReadWrite.All  |
+|Delegated (personal Microsoft account) |  Not supported.  |
+|Application | SecurityEvents.Read.All, SecurityEvents.ReadWrite.All |
+
+## HTTP request
+
+<!-- { "blockType": "ignored" } -->
+
+```http
+GET /security/alerts
+GET /security/alerts?$top=1
+GET /security/alerts?$filter={property} eq '{property-value}'
+GET /security/alerts?$filter={property} eq '{property-value}'&$top=5
+GET /security/alerts?$filter={property} eq '{property-value}'&{property} eq '{property-value}'
+```
+
+## Optional query parameters
+
+This method supports the following [OData query parameters](../../../concepts/query_parameters.md) to help customize the response:
+
+- `$count`
+- `$filter`
+- `$orderby`
+- `$select`
+- `$skip`
+- `$top`
+
+To return an alternative property set, use the OData `$select` query parameter to specify the set of **alert** properties that you want.  For example, to return the **assignedTo**, **category**, and **severity** properties, add the following to your query: `$select=assignedTo,category,severity`.
+
+## Request headers
+
+| Name      |Description|
+|:----------|:----------|
+| Authorization  | Bearer {code} (required)|

```

Comment:
For consistency, use the following construction for the Description for the Authorization header: Bearer {token}. Required.